packages/jest-matchers/src/matchers.js

```diff
@@ -398,6 +402,8 @@ const matchers: MatchersObject = {
 
     return {message, pass};
   },
+
+  toMatchSnapshot,
```

Comment:
Is there a reason these matchers were added in `jest-jasmine2` and not in `jest-matchers` ?